### DISCUSSION TITLE
Forgot/lost Admin Password

### DISCUSSION THREAD

**Discussion by MOOKBUFU** [chuot/rdio-scanner#413] [Q&A]
I’m certain there must be a way to recover my idiotic forgotten admin password- if anyone has some direction for such it would be greatly appreciated

**Comment by Thinline-Dynamic-Solutions** [comment]
Could try opening the database file in some editor. I’d make a copy of it
first and mess with the copy.

*Michael Chambers - Ohio Radio Scanner Network*
Founder/Manager/Support
***@***.**** ***@***.***>

**Reply by MOOKBUFU** [reply]
Thank you!

**Comment by jcrawford2000** [comment]
I know this is an old thread, but here's how you can reset a lost admin password.
You'll need to stop rdio-scanner while you do this.
Normal warning that you should backup the database before you go messing with it.
You'll need sqlite installed on your machine.  For Ubuntu:
  `sudo apt-get install sqlite`
Then you can run
  `sqlite3 rdio-scanner.db`
In sqlite, update the admin password to blank, which resets it back to the default:
  `update rdioScannerConfigs set val='' where key='adminpassword'`
Then exit sqlite:
   `.exit`
Start rdio-scanner back up, and you can log in with the default password.

**Comment by bburkett52** [comment]
Another way is to use the command line option listed in the pdf document
supplied with the release versions. Stop the program and restart with the
password change option and the new password . I don’t have that document in
front of me now to list the command.

~Bob

**Reply by jcrawford2000** [reply]
So, RTFM... I should have started there.  LOL!
Here's the excerpt from the docs:

```
[rdio@pc-linux rdio-scanner]$ ./rdio-scanner -h
Usage of ./rdio-scanner:
-admin_password string
change admin password

```

**Comment by bburkett52** [comment]
Always good to have a backup method! LOL!
~Bob
